firmware/src/network_model.c

```diff
@@ -181,7 +190,8 @@ static void ConfigureInterface(InterfaceState *interface) {
     interface->current_netmask = interface->configured_netmask;
     interface->config_source = CONFIG_SOURCE_STATIC;
   } else if (interface->configured_dhcp_mode &&
-             (dhcp_address = GetDHCPAddress())) {
+             (dhcp_address =
```

Comment:
I'm not personally a fan of this style of overloading, but I guess there isn't another sensible way to do it.